util/tournament_bracket.py

```diff
@@ -75,6 +77,25 @@ def build_bracket_tree(bracket_depth, players):
         
     return new_node
 
+def winrate(player_1, player_2, tag_rank, head_to_head):
+    p1_index = None
+    p2_index = None
+    tag1 = player_1["tag"].lower()
+    tag2 = player_2["tag"].lower()
+    if tag1 in tag_rank:
+        p1_index = tag_rank[tag1]
+    if tag2 in tag_rank:
+        p2_index = tag_rank[tag2]
+    if not p2_index:
+        return 1.0
+    if not p1_index:
+        return 0.0
+    p1_set_count = head_to_head[p1_index][p2_index]
+    p2_set_count = head_to_head[p2_index][p1_index]
+    p1_winrate = (0.5 / (1 + exp(-(player_1["seed"] - player_2["seed"]))) + 
```

Comment:
How do you know player 1 is higher or lower seed compared to player 2? The algorithm always assumes player 1 is the higher seeded player (e.g. 2nd seed compared to 5th seed), please update this function to reflect that by comparing the players' seeds at the top and assigning them accordingly.

In the 2/5 seed case, your logistic function should look like:
`... (0.5 / (1 + exp(-(player_2["seed"] - player_1["seed"]))) ...`, where player 2 is 5 seed, and player 1 is 2 seed.